config/environments/development.rb

```diff
@@ -63,6 +63,7 @@
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
 
   config.hosts << ".circulate.local"
+  config.hosts << "chicago-circulate.pdev"
```

Comment:
Did you run into any issues with the subdomain stuff? It’s pretty fresh and I’m sure there are sharp edges still.